src/utils.jsx

```diff
@@ -289,3 +289,71 @@ export function getFullUrl( uri ) {
   return parser.href
 }
 
+/**
+ * processes error messages so we can see them in redux
+ * @param {error} err the error object from api
+ * @returns {{name: string, message: string}} processed error object we can see
+ */
+export function processErrorMessage( err ) {
+  return {
+    name: err.name,
+    message: err.message
+  }
+}
+
+/**
+ * function to convert and compare 2 strings as dates
+ * @param {string} a input date string to compare MM/DD/YYYY or YYYY-MM-DD
+ * @param {string} b input date string to compare MM/DD/YYYY or YYYY-MM-DD
+ * @returns {boolean} lets us know if date is equal
+ */
+export function isDateEqual( a, b ) {
+  return new Date( a ).getTime() === new Date( b ).getTime()
```

Comment:
There might be moment functions that are better suited to this, like `isSame`

This function as written makes sure the epoch seconds are the same